### DISCUSSION TITLE
Migrate only user profile without C and D folders and files + tool to explore mig file.

### DISCUSSION THREAD

**Discussion by radzio6** [belowaverage-org/SuperGrate#100] [Ideas]
I have run first time Super Grate 1.4.0.0 and successfully migrated a user profile from one PC to another, however I noticed that beside user Profile also C  and D folders with data have been migrated. 
To verify I run [migviewer.exe](https://www.migee.com/2011/01/17/mig-recover-and-viewer-utility-alpha/) tool (created by Mike Morawski) and I noticed that C, D folders with files were  also backed up by Super Gate.
Is there a possibility to migrate only user profile without folder/files on other drives?

Also is there some other or newer tool to explore  USMT data file? 
Migrecover and viewer utility (migviewer.exe) is not being developed anymore?

**Comment by Deng-Xian-Sheng** [comment]
This can be implemented in USMT, but it is unknown whether this GUI wrapper has implemented it
